AEPEdgeMedia.xcodeproj/project.pbxproj

```diff
@@ -1725,6 +1750,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = FKGEE875K4;
```

Comment:
Hmm, should this be blank and not set to a specific value? Looks like this is the `debug` build for the `TestApp`.